integration/build.test.ts

```diff
@@ -0,0 +1,62 @@
+import {PrimerStyleDictionary} from '../src/PrimerStyleDictionary'
+import fs from 'fs'
+
+describe('PrimerStyleDictionary', () => {
+  const basePath = `./integration`
+  const extendedSD = PrimerStyleDictionary.extend({
+    source: [`${basePath}/tokens/**/*.json5`],
+    platforms: {
+      css: {
+        prefix: 'PREFIX',
+        transformGroup: 'css',
+        buildPath: `${basePath}/build/css/`,
+        files: [
+          {
+            options: {
+              showFileHeader: false,
+            },
+            destination: 'variables.css',
+            format: 'css/variables',
+          },
+        ],
+      },
+      advancedCss: {
+        prefix: 'PREFIX',
+        transformGroup: 'css',
+        buildPath: `${basePath}/build/css/`,
+        files: [
+          {
+            options: {
+              showFileHeader: false,
+            },
+            destination: 'advanced.css',
+            format: 'css/advanced',
+          },
+        ],
+      },
+    },
+  })
+
+  extendedSD.cleanAllPlatforms()
+  extendedSD.buildAllPlatforms()
+
+  it('it should transform with css/variables format', () => {
+    const output = fs.readFileSync(`${basePath}/build/css/variables.css`, 'utf8')
+    const expectedOutput = `:root {
+  --prefix-base-color-blue-500: #2C29FF; /* The primary color for interactive elements. */
+  --prefix-fg-color-link: #2C29FF;
```

Comment:
Similar question for this one.. the input is `fgColor` but output is `fg-color`. Is that normals? I'd expect it to also be `fgColor` in output?

Reply:
This is currently just testing the default naming transforms. Here this is expected behavior. I will add our custom name transformers and then this would be the expected behavior.

Reply:
ah okay.. so there's another transformer that does that, i think that makes sense then 👍